src/frontends/src/query_frontend_main.cpp

```diff
@@ -11,18 +11,21 @@ int main(int argc, char* argv[]) {
   // clang-format off
   options.add_options()
     ("redis_ip", "Redis address",
-        cxxopts::value<std::string>()->default_value("localhost"))
+        cxxopts::value<std::string>()->default_value(clipper::DEFAULT_REDIS_ADDRESS))
     ("redis_port", "Redis port",
-        cxxopts::value<int>()->default_value("6379"))
+        cxxopts::value<int>()->default_value(std::to_string(clipper::DEFAULT_REDIS_PORT)))
     ("threadpool_size", "Number of threads for the task execution threadpool",
-        cxxopts::value<int>()->default_value("4"));
+        cxxopts::value<int>()->default_value(std::to_string(clipper::DEFAULT_TASK_EXECUTION_THREADPOOL_SIZE)))
+    ("prediction_cache_size", "Size of the prediction cache in bytes, excluding cache metadata",
+       cxxopts::value<long>()->default_value(std::to_string(clipper::DEFAULT_PREDICTION_CACHE_SIZE_BYTES)));
```

Comment:
Done: added a `cache_size` argument to the constructor of `clipper_manager.Clipper`